harness/features/allVariables.local.test.ts

```diff
@@ -0,0 +1,89 @@
+import {
+    getConnectionStringForProxy,
+    forEachSDK,
+    createClient,
+    describeIf,
+    callAllVariables,
+    createUser,
+    wait,
+    mockServerUrl
+} from '../helpers'
+import { v4 as uuidv4 } from 'uuid'
+import { getServerScope } from '../mockServer'
+import { Capabilities, SDKCapabilities } from '../types'
+import { config, variables } from '../mockData'
+import nock, { Interceptor } from 'nock'
+
+jest.setTimeout(10000)
+
+describe('allVariables Tests - Local', () => {
+    const scope = getServerScope()
+    let configInterceptor: Interceptor
+
+    forEachSDK((name: string) => {
+        const capabilities: string[] = SDKCapabilities[name]
+        const clientId = uuidv4()
+        const sdkKey = `server_${clientId}`
+
+        let url: string
+
+        beforeAll(async () => {
+            configInterceptor = scope
+                .get(`/client/${clientId}/config/v1/server/${sdkKey}.json`)
+            configInterceptor
+                .reply(200, config)
+                .persist()
+
+            url = getConnectionStringForProxy(name)
+            await createClient(url, clientId, sdkKey, { baseURLOverride: `${mockServerUrl}/client/${clientId}` })
+            await wait(1000)
+        })
+
+        afterAll(() => {
+            nock.removeInterceptor(configInterceptor)
+        })
+
+        describeIf(capabilities.includes(Capabilities.local))(name, () => {
+            it('should return an empty object if client is not initialized', async () => {
```

Comment:
seems like this is more testing what happens when the SDK token is invalid?

Reply:
I didn't know how else to put the sdk in an uninitialized state. Got any suggestions?

Reply:
just make a new client then immediately call your method, that should do it